api/controllers/users.controller.js

```diff
@@ -0,0 +1,72 @@
+const db = require("../models");
+const { hashHelpers, jwtHelpers } = require('../helpers');
+const errors = require('../services/errorHandlers/index');
+
+const signup = async (req, res) => {
+
+    const emailExists = await db.users.findOne({ where: { email: req.body.email } });
+    const usernameExists = await db.users.findOne({ where: { username: req.body.username } });
```

Comment:
Are you sure you want to have unique username? I think user should have only unique email